apps/web/src/App.tsx

```diff
@@ -154,218 +151,216 @@ function App() {
-                            </SpotLight>
-                          </RequiredAuth>
-                        }
-                      />
-                      <Route
-                        path={ROUTES.TEAM}
-                        element={
-                          <RequiredAuth>
-                            <SpotLight>
-                              <MembersInvitePage />
-                            </SpotLight>
-                          </RequiredAuth>
-                        }
-                      />
-                      <Route
-                        path={ROUTES.CHANGES}
-                        element={
-                          <RequiredAuth>
-                            <SpotLight>
-                              <PromoteChangesPage />
-                            </SpotLight>
-                          </RequiredAuth>
-                        }
-                      />
-                      <Route
-                        path={ROUTES.SUBSCRIBERS}
-                        element={
-                          <RequiredAuth>
-                            <SpotLight>
-                              <SubscribersList />
-                            </SpotLight>
-                          </RequiredAuth>
-                        }
-                      />
-                      <Route
-                        path="/brand"
-                        element={
-                          <RequiredAuth>
-                            <SpotLight>
-                              <BrandPage />
-                            </SpotLight>
-                          </RequiredAuth>
-                        }
-                      />
-                    </Route>
-                  </Routes>
-                </SpotLightProvider>
-              </ThemeHandlerComponent>
-            </AuthHandlerComponent>
+                    <Route
+                      path={ROUTES.TEMPLATES_EDIT_TEMPLATEID}
+                      element={
+                        <RequiredAuth>
+                          <TemplateFormProvider>
+                            <TemplateEditorProvider>
+                              <SpotLight>
+                                <TemplateEditorPage />
+                              </SpotLight>
+                            </TemplateEditorProvider>
+                          </TemplateFormProvider>
+                        </RequiredAuth>
+                      }
+                    />
+                    <Route
+                      path={ROUTES.TEMPLATES}
+                      element={
+                        <RequiredAuth>
+                          <SpotLight>
+                            <NotificationList />
+                          </SpotLight>
+                        </RequiredAuth>
+                      }
+                    />
+                    <Route
+                      path="/general-started"
+                      element={
+                        <RequiredAuth>
+                          <SpotLight>
```

Comment:
Is it possible to wrap all logged in routes with those? so we won't have to do it manually, (where the layout is defined) I remember it was possible in previous versions.

Reply:
Not sure I follow what you suggest...

Reply:
Sorry for the confusion :P 

I meant that instead of copy pasting Those two provider wrappers on all routes individually, we can wrap them once at: https://github.com/novuhq/novu/pull/2851#discussion_r1111564308

this will affect all routes under the `<AppLayout/>` component